### DISCUSSION TITLE
[Schematics 實戰] 使用 Schematics 幫我們安裝指定套件 | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#91] [Announcements]
# [Schematics 實戰] 使用 Schematics 幫我們安裝指定套件 | 全端開發人員天梯

在開發專案時，通常不會全部都靠自己處理，有些功能會使用現成的套件，以免重複打造輪子的問題；若我們使用 Schematics 建立的程式有相依某些套件，則必須多一個安裝的步驟，手動確認處理的話實在是太辛苦了！其實，我們也可以透過 Schematics 來幫我們進行安裝套件的動作。今天我們來看看如何透過 Schematics 安裝指定的第三方套件到專案內！

https://fullstackladder.dev/blog/2019/12/22/schematics-workshop-06-install-packages/

**Comment by hildaChuang** [comment]
(雖然文章是 5 年前的了...) 在這段話「對於撰寫 task 的範例，可以參考 **NodePackageInstallTask 的程式碼**」的網址因為 branch 改名頁面已經不見了喔... :sweat:
